### PR TITLE
Unignored fixed testcase in bundle upMBdd_assign_role #1145

### PR DESCRIPTION
**Pull request by loro2**
Fix for issue #1145 

jcabi-github was already updated to version 0.41 in a previous issue, which allowed this code to be uncommented.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#2187](https://github.com/zerocracy/farm/issues/2187) is now in scope, role is `REV`

File: src/test/resources/com/zerocracy/bundles/upMBdd_assign_role/_after.groovy [hunk 1]
```diff
@@ -97,14 +98,9 @@ def exec(Project project, XML xml) {
...
-  //  When the bug is fixed, update jcabi-github and uncomment code below.
-//  MatcherAssert.assertThat(
-//    'DEV role was assigned to anonymous',
-//    roles.allRoles('anonymous'),
-//    Matchers.emptyIterable()
-//  )
+  MatcherAssert.assertThat(
+    'DEV role was assigned to anonymous',
+    roles.allRoles('anonymous'),
+    Matchers.emptyIterable()
```
Comment (paulodamaso) [inline src/test/resources/com/zerocracy/bundles/upMBdd_assign_role/_after.groovy]: @loro2 Let's just  make this assertion more elegant, please use `new IsEmptyIterable()` instead `Matchers.emptyIterable()`

Comment (loro2) [reply src/test/resources/com/zerocracy/bundles/upMBdd_assign_role/_after.groovy]: Done

Comment (loro2) [reply src/test/resources/com/zerocracy/bundles/upMBdd_assign_role/_after.groovy]: One more commit, removed the surplus diamond, now all should be ok

Comment (loro2) [reply src/test/resources/com/zerocracy/bundles/upMBdd_assign_role/_after.groovy]: Ok, one more commit, a hanging import was left there

**Comment by g4s8** [comment]
@loro2 please name your PRs as described here: https://www.zerocracy.com/policy.html#42
> Pull request title explains the problem it is fixing;

**Comment by loro2** [comment]
> 
> 
> @loro2 please name your PRs as described here: https://www.zerocracy.com/policy.html#42
> 
> > Pull request title explains the problem it is fixing;

@g4s8 done

**Comment by paulodamaso** [comment]
@g4s8 Looks good for me, please merrge

**Comment by g4s8** [comment]
@rultor merge

**Comment by loro2** [comment]
@g4s8 merged fixed master into feature branch

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
Code review was too long (25 days), architects (@g4s8) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#2187](https://github.com/zerocracy/farm/issues/2187) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by sereshqua** [comment]
@loro2 please make sure you start all your comments with the name of the user they are referred to, [see](https://github.com/zerocracy/farm/pull/2187#discussion_r339292707)

@paulodamaso please make sure you try to find at least 3 issues during next CR

**Comment by loro2** [comment]
@sereshqua confirmed

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Pull request merged**